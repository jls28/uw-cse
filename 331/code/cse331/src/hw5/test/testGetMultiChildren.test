CreateGraph g1
AddNode g1 n1
AddNode g1 n2
AddEdge g1 n1 n2 COOLEDGE
AddEdge g1 n1 n2 COOLEDGE2
AddEdge g1 n1 n2 COOLEDGE3
ListChildren g1 n1