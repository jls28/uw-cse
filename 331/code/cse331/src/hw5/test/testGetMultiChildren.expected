created graph g1
added node n1 to g1
added node n2 to g1
added edge COOLEDGE from n1 to n2 in g1
added edge COOLEDGE2 from n1 to n2 in g1
added edge COOLEDGE3 from n1 to n2 in g1
the children of n1 in g1 are: n2(COOLEDGE) n2(COOLEDGE2) n2(COOLEDGE3)
